### PR TITLE
Feature/activities component

### LINKED ISSUE #10: Activities component architecture

### PR DESCRIPTION
**Pull request by KatherineBlaine**
### Files Changed:

index.html: imported google fonts
Activities.css: styling for activities component
Activities.js: functionality for activities component
App.js: pass activities state prop to rendering of Activities component, update app state - Activity prop. Update on click of 'go' button
Header.css: styling for Beam header
Header.js: update header from test data

### What Has Changed: 

**Activities.js**
- Pass activities array from App state
- Install and implement useState hook to hold on to the index position of the current activity being displayed
- Conditionally render arrow buttons when all activities have been viewed
- Arrow buttons update Activities state by adding/subtracting from the value of the current index, and re-render the page based on that value

The remainder of changes were styling/imports!

### Notes: (includes questions, ideas for the future, anything you want to call out)

- I considered putting 'prevActivity' and 'nextActivity' into a single function with conditional logic. Could consolidate the component structure a bit but not sure if it is necessary, let me know what you think!
- Some styling is in px which isn't ideal for responsiveness so that is an opportunity for refactoring.

### Closes: (any issues that it closes)

closes #10

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,6 +3,10 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Playfair+Display+SC:ital,wght@1,700&family=Plus+Jakarta+Sans&display=swap" rel="stylesheet">
```
Comment (espressoGoddess) [inline public/index.html:9]: excited to see what these fonts look like

File: src/Components/Activities/Activities.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  background-color: white;
+  font-family: 'Plus Jakarta Sans', sans-serif;
+}
+
+.activity-page {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (jfogiato) [inline src/Components/Activities/Activities.css:43]: Loving the styling!!

File: src/Components/Activities/Activities.js [hunk 3]
```diff
@@ -1,13 +1,32 @@
-import React from "react";
+import React, { useState } from "react";
+import './Activities.css';
+
+const Activities = ({ activities, updateActivity }) => {
+  const [currentActivityIndex, setCurrentActivityIndex] = useState(0);
+
+  const nextActivity = () => {
+    let nextIndex = currentActivityIndex + 1;
```
Comment (jfogiato) [inline src/Components/Activities/Activities.js:8]: Should likely switch these to `const`'s are some point!

File: src/Components/Activities/Activities.js [hunk 4]
```diff
@@ -1,13 +1,32 @@
...
-    </div>
-    <button>Go</button>
-    </>
+    <section className="activity-page">
+      <div className="activity-card">
+        {currentActivityIndex > 0 && <button onClick={() => prevActivity()}>←</button>}
+        <h2>{activities[currentActivityIndex].activity}</h2>
+        {currentActivityIndex < activities.length - 1 && <button onClick={() => nextActivity()}>→</button>}
+      </div>
+      <button id="go-button" onClick={() => selectActivity()}>Let's do it!</button>
```
Comment (jfogiato) [inline src/Components/Activities/Activities.js:28]: Can simplify the 3 activity methods here to just be the method names sans anon functions!

File: src/Components/Activities/Activities.js [hunk 5]
```diff
@@ -1,13 +1,32 @@
...
-    <div>
-      <h2>Go find a spider!</h2>
-    </div>
-    <button>Go</button>
-    </>
+    <section className="activity-page">
+      <div className="activity-card">
+        {currentActivityIndex > 0 && <button onClick={() => prevActivity()}>←</button>}
+        <h2>{activities[currentActivityIndex].activity}</h2>
+        {currentActivityIndex < activities.length - 1 && <button onClick={() => nextActivity()}>→</button>}
```
Comment (espressoGoddess) [inline src/Components/Activities/Activities.js:26]: would love to chat about this, not quite understanding it

File: src/Components/Activities/Activities.js [hunk 6]
```diff
@@ -1,13 +1,32 @@
-import React from "react";
+import React, { useState } from "react";
+import './Activities.css';
+
+const Activities = ({ activities, updateActivity }) => {
+  const [currentActivityIndex, setCurrentActivityIndex] = useState(0);
```
Comment (espressoGoddess) [inline src/Components/Activities/Activities.js:5]: so state is holding the current index basically?

**Review by jfogiato — APPROVED** [review]

**Review by espressoGoddess — APPROVED** [review]

File: src/Components/Header/Header.css [hunk 7]
```diff
@@ -0,0 +1,6 @@
+h1 {
+  font-family: 'Playfair Display SC', serif;
```
Comment (chrissycooper) [inline src/Components/Header/Header.css:2]: just want to say, I only added one font for an accent style and I am not at all committed to it, so if you like any fonts I am down for it!

File: src/Components/Activities/Activities.js (hunk 5 above)
Comment (chrissycooper) [inline src/Components/Activities/Activities.js:26]: nice work, I like the arrow buttons, and I do think this is super straightforward and easy to follow conditional rendering. Appreciate ya!

**Review by chrissycooper — APPROVED** [review]

**Pull request merged**